package.json

```diff
@@ -58,6 +60,7 @@
   },
   "peerDependencies": {
     "react": "^16.8.6",
-    "react-dom": "^16.8.6"
+    "react-dom": "^16.8.6",
```

Comment:
`react@^16.3.0` ?
`react-dom@^16.3.0` ?

Reply:
Thanks for reminding. I use `^16.3.0` for `peerDependencies` and lock the version to `16.8.4` in `devDependencies` according to the `yarn.lock` of `twreporter-react`
https://github.com/twreporter/twreporter-react/blob/master/yarn.lock#L7377